src/auto_scheduler/transform_step.cc

```diff
@@ -923,5 +958,275 @@ String ComputeRootStepNode::PrintAsPythonAPI(Array<te::Stage>* stages,
   return ss.str();
 }
 
+/********** Primitives adding new stages **********/
+
+/*!
+ * \brief Common part for steps that add new stages(e.g. CacheReadStep, CacheWriteStep,
+ * RfactorStep). This will filter out all steps that can change the number of stages in a
```

Comment:
```suggestion
 * RfactorStep). This returns all steps that can change the number of stages in a
```
"Filter out" is the antonym of "return"